docs/dashboard/import-export.md

```diff
@@ -0,0 +1,29 @@
+# Altis Dashboard Import Export
+
+The Altis Dashboard allows you to perform data migrations between environments, as well as perform data exports for use in local development environments.
+
+As shown below, you can find the Import & Export features under the Data tab of the selected Altis Environment.
+
+![import export view](../assets/import-export-view.png)
+
+From here you can view and download previously taken Exports, displayed as a list, ordered by date. 
+
+You can also create a new Export of your Database and/or Assets. Typically this would be comprised of your WordPress Uploads directory, in addition to any other assets your application may be uploading to your Uploads directory.)
+
+To create a new Export, you can press the Export button. This will display a new modal (as shown below) in which you can choose to export the uploads and/or the database. If you choose to create an export of the Uploads, you’ll be able to optionally specify a path. This is useful if you’d quickly like to create an export of assets in a specific directory.
+
+![export modal](../assets/export-modal.png)
+
+Via the Altis Dashboard, you can also Import your database and assets from one environment to another. 
```

Comment:
You can also import your database and assets from one environment to another, via the Altis Dashboard.